lib/beaker/dsl/helpers/host_helpers.rb

```diff
@@ -256,13 +256,21 @@ def rsync_to host, from_path, to_path, opts = {}
         #                               'archive/sut-files'.
         #
         # @return [Result] Returns the result of the #scp_from operation.
-        def archive_file_from(host, from_path, opts = {}, archive_root = 'archive/sut-files')
+        def archive_file_from(host, from_path, opts = {}, archive_root = 'archive/sut-files', archive_name = 'sut-files.tgz')
```

Comment:
`archive_name` doesn't have yard docs yet; looks like it should be similar to the `archive_root` documentation.